hack/local-csi-driver-charts-sync/local-csi-driver-charts.diff

```diff
@@ -1,5 +1,5 @@
 diff --git a/./local-csi-driver-charts.yaml b/./charts/kaito/workspace/templates/local-csi-driver-ds.yaml
-index 93d0b57..54cb7e4 100644
```

Comment:
why there is a diff file here?

Reply:
local-csi-driver is an individual repo which have there own helm charts. https://github.com/Azure/local-csi-driver/tree/main/charts/latest

It's a part of the kaito helm chart here. Thus I add a diff file to maintain this chart, sync from original repo.
manifests rendered by helm will have some specific label which we don't want to include here.